public/models/README.md

```
@@ -0,0 +1,84 @@
+
+## Integration
+
+`SpeakerDiarizer` (`public/app/speaker-diarizer.js`) expects three pre-loaded `ort.InferenceSession` instances.
+Run diarization **after** the Octa-Pass STFT → ML → iSTFT pipeline on the **cleaned** buffer for best accuracy.
+
+```javascript
+import SpeakerDiarizer from './speaker-diarizer.js';
+
+const diarizer = new SpeakerDiarizer(segSession, embSession, vadSession, 16000);
+const timeline = await diarizer.diarize(processedAudioBuffer);
+```
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)